lib/pbench/test/unit/server/conftest.py

```diff
@@ -152,7 +153,15 @@ def client(server_config, fake_email_validator):
 
 
 @pytest.fixture()
-def db_session(server_config):
+def make_logger(server_config):
+    """
+    Construct a Pbench Logger object
+    """
+    return get_pbench_logger("TEST", server_config)
```

Comment:
Do we really want to call `get_pbench_logger()` for this?  It's a fairly complicated function, and it requires a config file, which is not very "unit-like".

Can't we just call `logging.getLogger()` and maybe apply `_StyleAdapter` to what it returns?

Reply:
This is easier, more compatible, and more importantly I only moved an existing fixture from a specific test to the `conftest.py`. Getting the right formatter is always the tricky part, and it's easiest to just use the mechanism we've already got.